apps/heuritech-website-e2e/src/integration/common/api.js

```diff
@@ -3,6 +3,33 @@
 import { Given, When } from 'cypress-cucumber-preprocessor/steps';
 
 Given('I use the mocked api', () => {
+  cy.intercept(
+    {
+      url: 'http://localhost:3333/trends*',
+      method: 'GET',
+      query: { page: 1, page_size: 20 }
+    },
+    { fixture: 'trends.json' }
+  ).as('baseTrends');
+
+  cy.intercept(
+    {
+      url: 'http://localhost:3333/trends*',
+      method: 'GET',
+      query: { page: 2, page_size: 20 }
+    },
+    { fixture: 'trendsPageTwo.json' }
+  ).as('trendsPageTwo');
+
+  cy.intercept(
+    {
+      url: 'http://localhost:3333/trends*',
+      method: 'GET',
+      query: { page: 3, page_size: 20 }
+    },
+    { fixture: 'trendsPageThree.json' }
+  ).as('trendsPageThree');
```

Comment:
ℹ️ There is probably a better way to get fixtures based on a dynamic query, leaving this to a further dedicated PR